elements/pl-multiple-choice/pl-multiple-choice.py

```diff
@@ -70,6 +70,24 @@ def categorize_options(element, data):
     return correct_answers, incorrect_answers
 
 
+def generate(element_html, data):
+    element = lxml.html.fragment_fromstring(element_html)
+    correct_answers, incorrect_answers = categorize_options(element, data)
+
+    len_all = len(correct_answers + incorrect_answers)
+    len_unique = len(set(correct_answers + incorrect_answers))
+
+    # Making a conscious choice *NOT* to apply .lower() to all list elements
+    # in case instructors want to explicitly have matrix M vs. vector m as
+    # possible options.
+
+    if len_unique < len_all:
+        raise Exception(
+            f'pl-multiple-choice element cannot have duplicate choices. For e.g., this choice-set: \
+                {sorted(correct_answers + incorrect_answers)} has {len_all-len_unique} duplicate(s).'
```

Comment:
Instead of just detecting that there *are* duplicates, what do you think about actually computing the duplicates and changing the error message?
```suggestion
            f'pl-multiple-choice element has duplicate choices: {', '.join(duplicates)}'
```

Reply:
Sure, I could do that. 

It'll probably be a slower operation though. At least that bit of code will only run when there are duplicates. 

Ok, on it!